pylint_venv.py

```diff
@@ -108,7 +108,7 @@ def activate_venv(venv):
     sys.path[:] = new_paths + kept_paths
 
 
-def inithook(venv=None, force_venv_activation=False):
+def inithook(venv=None, force_venv_activation=False, quiet=False):
```

Comment:
```suggestion
def inithook(venv=None, force_venv_activation=False, **, quiet=False):
```
I would make this a keyword only argument because it gives more flexibility in the future for introducing other arguments without breaking change in the future, and passing in `quiet=True` will be clearer than just passing in `True`.

Reply:
Of course, ideally `force_venv_activation` would also be a keyword only argument. However, this would be a breaking change at this point.